examples/ssao/main.js

```diff
@@ -0,0 +1,701 @@
+( function () {
+    'use strict';
+
+    var ExampleOSGJS = window.ExampleOSGJS;
+
+    var OSG = window.OSG;
+    var osg = OSG.osg;
+    var osgViewer = OSG.osgViewer;
+    var osgShader = OSG.osgShader;
+    var osgUtil = OSG.osgUtil;
+    var Texture = osg.Texture;
+
+    var $ = window.$;
+    var P = window.P;
+
+    // Generates MAX_MIP_LEVEL depth mipmaps.
+    // Do not forget to change the code in ssaoFragment.glsl
+    // because the mipmaps are passed manually using texture units
+    var MAX_MIP_LEVEL = 5;
+
+    var shaderProcessor = new osgShader.ShaderProcessor();
+
+    // Loads a glTF model and update
+    // the current displayed scene in the Example
+    var loadFiles = function ( example, files, sceneName ) {
+
+        var promise = OSG.osgDB.Registry.instance().getReaderWriterForExtension( 'gltf' )
```

Comment:
Hey let me chime in. The normal usage for loading nodes should be 
`osgDB.readNodeURL(` 'nodeURL' );`

The reader parse calls the registry for you: 

https://github.com/cedricpinson/osgjs/blob/master/sources/osgDB/readerParser.js#L34